server/src/main/java/org/cloudfoundry/identity/uaa/zone/MultitenantJdbcClientDetailsService.java

```diff
@@ -36,17 +47,10 @@
 import org.springframework.util.Assert;
 import org.springframework.util.StringUtils;
 
-import javax.sql.DataSource;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Timestamp;
-import java.util.*;
-
 /**
  * A copy of JdbcClientDetailsService but with IdentityZone awareness
  */
-public class MultitenantJdbcClientDetailsService implements ClientServicesExtension,
```

Comment:
`MultitenantClientDetailsService` is a single interface that extends all three of these interfaces and was necessary for the fix to `HomeController` above.